backend/medtagger/api/app.py

```diff
@@ -11,18 +11,19 @@
 # pylint: disable=unused-import;  It's used by Flask
 from flask import Flask, current_app
 from flask_cors import CORS
+from flask_security import Security, SQLAlchemyUserDatastore
+from medtagger.database.models import User, Role
```

Comment:
Please move this import to line 18. More about Python import grouping/order: https://stackoverflow.com/a/20763446/2839291

Reply:
Fixed